src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -524,7 +523,8 @@ def estimateTime(self, remaining=None):
 
         stitch_time = (self._nx * self._ny * max_pxs * self._overlap) / self.STITCH_SPEED
         try:
-            move_time = max(self._guessSmallestFov(self._streams)) * (remaining - 1) / self._move_speed
+            # Extra time to account for actual stage movement happening in more than 2 dimensions
```

Comment:
What do you mean by "more than 2 dimensions"? Isn't stage move always just X&Y?

Reply:
The meteor stage moves in two dimensions x and y. As y is linked to y and z of the stage-bare, the movement in stitching the tiles occurs in 3 dimensions.